src/tribol/physics/Physics.hpp

```diff
@@ -11,8 +11,8 @@
 namespace tribol {
 
 // forward declarations
-struct InterfacePair;
-struct SurfaceContactElem;
+// struct InterfacePair;
+// struct SurfaceContactElem;
```

Comment:
Can these just be removed?  Are they not used anywhere in the header?

Reply:
These are now removed.